src/mono/wasm/runtime/startup.ts

```diff
@@ -234,8 +234,15 @@ async function onRuntimeInitializedAsync(userOnRuntimeInitialized: () => void) {
         await runtimeHelpers.afterPreRun.promise;
         mono_log_debug("onRuntimeInitialized");
 
-        runtimeHelpers.mono_wasm_exit = cwraps.mono_wasm_exit;
+        runtimeHelpers.mono_wasm_exit = (exit_code) => {
+            if (ENVIRONMENT_IS_NODE && !ENVIRONMENT_IS_WORKER) {
+                Module.runtimeKeepalivePop();
```

Comment:
Is it wise to do this before calling mono_wasm_exit instead of after? Or in a finally block?

Reply:
mono_wasm_exit on nodejs will call node's `exit()` and stop the node process when `runtimeKeepalivePop` balances. So it need be called before.